app/pnpm-workspace.yaml

```diff
@@ -0,0 +1,6 @@
+ignoredBuiltDependencies:
+  - better-sqlite3
```

Comment:
How does this work, doesn't it need the build script to fetch the prebuilt binary?